src/components/api/codenvy-factory.factory.js

```diff
@@ -152,7 +152,10 @@ class CodenvyFactory {
 
       let seeLink = [];
       this.lodash.find(tmpFactory.links, function (link) {
-        if (link.rel === 'create-workspace') {
+        if (link.rel === 'accept') {
+          seeLink.push(link.href);
+        }
```

Comment:
strange to use two if blocks
WDYT about
if ('accept' === link.rel || 'accept-named' === link.rel) {